sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/src/Framework/AuthenticationEventMetadata.cs

```diff
@@ -71,7 +72,16 @@ internal AuthenticationEventRequestBase CreateEventRequest(HttpRequestMessage re
 
             if (validate)
             {
-                Helpers.ValidateGraph(eventRequest);
+                // Validate the request body. If the validation fails, throw a request validation
+                // exception that will return a 500 error back to eSTS.
+                try
+                {
+                    Helpers.ValidateGraph(eventRequest);
+                }
+                catch (ValidationException exception)
+                {
+                    throw new RequestValidationException(exception.Message);
```

Comment:
I'd suggest embedding the original exception as an inner exception here.

Reply:
Done